sprint6/DO_NOT_TOUCH/zadanie3.js

```diff
@@ -0,0 +1,16 @@
+/*
+Zadanie 3. Napisz funkcję, która wypisze wszystkie liczby od 0 do 100
+*/
+
+function writeNumber() {
```

Comment:
Funkcja niech odzwierciedla to za co funkcja jest odpowiedzialna, w tym przypadku np. `wypiszLiczbyDo100`